app/src/main/java/at/roteskreuz/stopcorona/model/repositories/HandshakeCodewordRepository.kt

```diff
@@ -0,0 +1,34 @@
+package at.roteskreuz.stopcorona.model.repositories
+
+import androidx.annotation.StringRes
+import at.roteskreuz.stopcorona.model.repositories.other.ContextInteractor
+
+/**
+ * Repository to convert a random number into a readable identification word.
+ */
+interface HandshakeCodewordRepository {
+
+    /**
+     * Returns the related codeword for a random number.
+     */
+    fun getCodeword(identification: String): String?
```

Comment:
Input parameter should be specified (limited). I suggest to use `@IntRange(from = 0, to = 9999) identification : Int` and in implementation convert it to string and prefix by zeros to have length equal 4. Then your code is safe for inputs and have expected output.

Or at least specify input string in KDoc.